app/services/control_rate_service.rb

```diff
@@ -60,11 +63,25 @@ def fetch_all_data
     results
   end
 
+  def registration_counts_with_exclusions
+    return @registration_counts_with_exclusions if defined? @registration_counts_with_exclusions
+    formatter = lambda { |v| quarterly_report? ? Period.quarter(v) : Period.month(v) }
+
+    @registration_counts_with_exclusions =
+      region.assigned_patients
+        .for_reports(with_exclusions: with_exclusions)
```

Comment:
If `with_exclusions` is false, this will duplicate what we are already storing in registrations, right?

Reply:
This applies the with_hypertension clause in that case, which I think we should've been adding already to the denominator.